File: b186(computer_science)/C++/computer_science.cpp
#include <iostream>
#define min(a, b) (a < b ? a : b)

int main(void) {
  long long cookies, chocolates, cakes;
  while (std::cin >> cookies >> chocolates >> cakes) {
    chocolates += min(cookies / 10, cakes / 2);
    std::cout << cookies << " 個餅乾，" << chocolates << " 盒巧克力，" << cakes << " 個蛋糕。\n";
  }
  return 0;
}

File: b186(computer_science)/Python/computer_science.py
from sys import stdin, stdout
while True:
    line = stdin.readline()
    if not line: break
    cookies, chocolates, cakes = map(int, line.split())
    chocolates += min(cookies // 10, cakes // 2)
    stdout.write(f'{cookies} 個餅乾，{chocolates} 盒巧克力，{cakes} 個蛋糕。\n')
